app/controllers/web/console.php

```diff
@@ -390,6 +390,7 @@
             ->setParam('fileLimit', App::getEnv('_APP_STORAGE_LIMIT', 0))
             ->setParam('fileLimitHuman', Storage::human(App::getEnv('_APP_STORAGE_LIMIT', 0)))
             ->setParam('timeout', (int) App::getEnv('_APP_FUNCTIONS_TIMEOUT', 900))
+            ->setParam('usageStatsEnabled',App::getEnv('_APP_USAGE_STATS','enabled') == 'enabled');
```

Comment:
I don't think you need access to this value in the page layout itself. The `$page` component should be enough.

Reply:
@eldadfux Don't understand what you mean

Reply:
Have not passed in the layout, i have passed in the $page component, some hidden lines in the middle made you think I am passing the param in the layout :)

Reply:
Sorry, my bad :)